tests/cpu/pmp_noexec_ram/main.c

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2023 Bennet Blischke <bennet.blischke@haw-hamburg.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief Test application for the pmp_noexec_ram pseudo-module
+ *
+ * @author Sören Tempel <tempel@uni-bremen.de>
+ * @author Bennet Blischke <bennet.blischke@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdint.h>
+
+#include "cpu.h"
+#include "pmp.h"
+
+#define JMPBUF_SIZE 3
+
+int main(void)
+{
+    uint32_t buf[JMPBUF_SIZE];
+
+    /* Fill the buffer with invalid instructions */
+    for (unsigned i = 0; i < JMPBUF_SIZE; i++) {
+        buf[i] = UINT32_MAX;
+    }
```

Comment:
heh, so wouldn't this crash either way? Can we fill it with no-op?

Reply:
Certainly, the tests works by ensuring `MEM MANAGE HANDLER` is in the crash. If it crashes due to invalid instruction, the output will be `unhandled trap` or similar.

Reply:
I think I will leave it as `uint32_max`. In the near future, the test will be integrated with the MPU test. Having a generic crash values simplifies that.